src/audio/copier/copier.h

```diff
@@ -291,4 +298,8 @@ enum sof_ipc_stream_direction
 
 void copier_update_params(struct copier_data *cd, struct comp_dev *dev,
 			  struct sof_ipc_stream_params *params);
+
+#if CONFIG_MICROPHONE_PRIVACY
+void mic_privacy_event(void *arg, enum notify_id type, void *data);
+#endif
```

Comment:
Why you declare the event here in the header file while the `mic_privacy_configure()` is in the source file and both is only used in there?
And the `mic_privacy_event()` does not even needs forward declaration as it is used on the configure function.

Reply:
No idea why it was there, removed